lms/static/js/groups/views/cohort_editor.js

```diff
@@ -18,11 +27,38 @@
         render: function() {
             this.$el.html(this.template({
                 cohort: this.model,
-                advanced_settings_url: this.advanced_settings_url
+                contentGroups: this.contentGroups,
+                studioAdvancedSettingsUrl: this.context.studioAdvancedSettingsUrl
             }));
+            this.cohortFormView = new CohortFormView({
+                model: this.model,
+                contentGroups: this.contentGroups,
+                context: this.context
```

Comment:
Just pass this.context.studioGroupConfigurationsUrl to the view (which is consistent with what you do on line 31)?